### PR TITLE
Add essay on animal testing

### PR DESCRIPTION
**Pull request by salutis**
⚠️ With this one, I did a quick, 1-minute research before writing. ⚠️

### CODE REVIEW

File: Animal Testing.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Animal Testing
+
+I am sure we can all agree, and the modern research backs this up, that animals do feel pain. Yet, for some reason, when it comes to performing, often painful, experiments on them, many of us close their eyes. Let us look at how this is not our only option and explore some of the alternatives available today.
+
+The scientists rarely need the entire body of an animal to carry out their experiments. In fact, much of the experimentation can be conducted on colonies of cells which, owing to advancements in modern biology, can be reliably isolated and used for this purpose. These, so called, in vitro experiments are slowly replacing unnecessary experimentation on animals in many modern laboratories all around the world.
+
+Another option gaining popularity is the use of computer simulations. With modern hardware being more powerful than ever as well as significant advancements in study of algorithms, we are now able simulate complex systems such as living cells and simple organisms. This, so called, in silicon approach is currently being intensively studied and explored by various cutting edge laboratories.
```
[the remark this answers is not in the document]
Comment (mikemee) [reply Animal Testing.txt]: put quotes around "in silicon"

**Comment by salutis** [comment]
Review feedback addressed. Thank you @mikemee!

File: Animal Testing.txt [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+I am sure we can all agree, and the modern research backs this up, that animals do feel pain. Yet, for some reason, when it comes to performing, often painful, experiments on them, many of us close our eyes. Let us look at how this is not our only option and explore some of the alternatives available today.
 
-The scientists rarely need the entire body of an animal to carry out their experiments. In fact, much of the experimentation can be conducted on colonies of cells which, owing to advancements in modern biology, can be reliably isolated and used for this purpose. These, so called, in vitro experiments are slowly replacing unnecessary experimentation on animals in many modern laboratories all around the world.
+The scientists rarely need the entire body of an animal to carry out their experiments. In fact, much of the experimentation can be conducted on colonies of cells which, owing to advancements in modern biology, can be reliably isolated and used for this purpose. These, so called, "in vitro" experiments are slowly replacing unnecessary experimentation on animals in many modern laboratories all around the world.
```
Comment (mikemee) [inline Animal Testing.txt]: Scientists rarely need... (Omit the leading "The ")

File: Animal Testing.txt [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 
-Another option gaining popularity is the use of computer simulations. With modern hardware being more powerful than ever as well as significant advancements in study of algorithms, we are now able simulate complex systems such as living cells and simple organisms. This, so called, in silicon approach is currently being intensively studied and explored by various cutting edge laboratories.
+Another option gaining popularity is the use of computer simulations. With modern hardware being more powerful than ever, as well as significant advancements in study of algorithms, we are now able simulate complex systems such as living cells and simple organisms. This, so called, "in silicon" approach is currently being intensively studied and explored by various cutting edge laboratories.
```
Comment (mikemee) [inline Animal Testing.txt]: in _the_ study of algorithms.  _Or more concisely "as well as significant advances in algorithms"._

Comment (mikemee) [reply Animal Testing.txt]: Missed this: in **the** study of algorithms

File: Animal Testing.txt (hunk 2 above)
Comment (mikemee) [reply Animal Testing.txt]: Didn't remove **The** ....

**Comment by salutis** [comment]
@mikemee 

> Comments not addressed?

See above, addressed in #10.

**Pull request merged**